spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -32,6 +32,14 @@ def reset_password_instructions
     )
   end
 
+  def reset_password_instructions_with_pending_gpo_letter
```

Comment:
Since this is being added in another pr it seems like we can remove it from here.

Reply:
This no longer appears in the diff since the change was merged into main.